pages.fr/sunos/snoop.md

```diff
@@ -0,0 +1,25 @@
+# snoop
+
+> Renifleur de paquets réseau.
+> Équivalent SunOS de tcpdump.
+> Plus d'information : <https://www.unix.com/man-page/sunos/1m/snoop>.
+
+- Capturer des paquets sur une interface réseau spécifique :
+
+`snoop -d {{e1000g0}}`
+
+- Enregistrer les paquets capturés dans un fichier au lieu de les afficher :
+
+`snoop -o {{filename}}`
+
+- Afficher le résumé détaillé de la couche de protocole des paquets d'un fichier :
+
+`snoop -V -i {{filename}}`
+
+- Capturez les paquets réseau provenant d'un nom d'hôte et accédez à un port donné :
+
+`snoop to port {{port}} from host {{hostname}}`
```

Comment:
Please translate the placeholders, like `hostname`, `port`, `id_address`, etc., for get this easier for non-English speakers.